core/jvm/src/test/scala/zio/sql/ProductSchema.scala

```diff
@@ -10,17 +13,24 @@ object ProductSchema {
 
     override def renderInsert[A: Schema](insert: self.Insert[_, A]): String = ???
   }
-  import sql.ColumnSet._
+
   import sql._
 
-  val productTable = (
-    string("id") ++
-      localDate("last_updated") ++
-      string("name") ++
-      int("base_amount") ++
-      int("final_amount") ++
-      boolean("deleted")
-  ).table("product")
```

Comment:
We should still have tests that are using the "old way".

Reply:
We are not supporting ++ for definition of "source" tables. Let's discuss if there's a use case for having both.